compiler/jsgen.nim

```diff
@@ -2406,8 +2406,7 @@ proc genCast(p: PProc, n: PNode, r: var TCompRes) =
     r.res = "($1 $2)" % [r.res, trimmer]
   elif toInt:
     if fromInt:
-      let trimmer = unsignedTrimmer(dest.size)
-      r.res = "($1 $2)" % [r.res, trimmer]
```

Comment:
I guess we need: r.res = "($1|0)" % [r.res] 
though possibly not very important.

Araq's preference would be to use `discard` instead of `return` in this case.

Reply:
there's if dest.kind == src.kind: return , so I take that style. and there's nothing else to do , return is fine to me.